public/plugin.tsx

```diff
@@ -32,6 +32,9 @@ import {
   observabilityApplicationsID,
   observabilityApplicationsPluginOrder,
   observabilityApplicationsTitle,
+  observabilityGettingStartedID,
+  observabilityGettingStartedTitle,
+  observabilityGettingStartedPluginOrder,
```

Comment:
I see this in the notebooks PR too. Can we remove them from here?

Reply:
Done with update.